src/main/java/com/example/java/test/junior/developer/dto/LogOutResponseDto.java

```diff
@@ -0,0 +1,24 @@
+package com.example.java.test.junior.developer.dto;
+
+import javax.validation.constraints.NotBlank;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
```

Comment:
It should be `@Value`, not `@Data` because we should keep immutable everything that can be immutable.